app/lib/admin/backend.dart

```diff
@@ -379,15 +379,16 @@ class AdminBackend {
       final Key packageKey = _db.emptyKey.append(Package, id: packageName);
       final package = await tx.lookupOrNull<Package>(packageKey);
       if (package == null) {
-        print('Package $packageName does not exist.');
+        throw Exception(
+            'Package "$packageName" does not exists. Use full package removal without the version qualifier.');
```

Comment:
What does using full package removal help?

Reply:
It will run regardless of the `Package` entity existing. Removing single version assumes an existing `Package` entity, as it needs to modify entries on it.